### PR TITLE
Renamed "Google My Business/GMB" to "Business Profile" 

### PR DESCRIPTION
**Pull request by fiboknacky**

### CODE REVIEW

File: examples/Utils/ArgumentNames.php [hunk 1]
```diff
@@ -171,7 +171,7 @@ final class ArgumentNames
         self::GCLID => 'The Google Click ID',
         self::GEO_TARGET_CONSTANT_ID => 'The geo target constant ID',
         self::GMB_ACCESS_TOKEN => 'The access token used for uploading GMB location feed data',
```
Comment (PierrickVoulet) [inline examples/Utils/ArgumentNames.php:173]: `GMB_ACCESS_TOKEN` should  be  changed to `BUSINESS_PROFILE_ACCESS_TOKEN`

File: examples/Utils/ArgumentNames.php [hunk 2]
```diff
@@ -75,8 +75,8 @@ final class ArgumentNames
     public const FREEFORM_KEYWORD_TEXT = 'freeformKeywordText';
     public const GCLID = 'gclid';
     public const GEO_TARGET_CONSTANT_ID = 'geoTargetConstantId';
-    public const GMB_ACCESS_TOKEN = 'gmbAccessToken';
-    public const GMB_EMAIL_ADDRESS = 'gmbEmailAddress';
+    public const BUSINESS_PROFILE_ACCESS_TOKEN = 'businessProfileAccessToken';
+    public const BUSINESS_PROFILE_EMAIL_ADDRESS = 'businessProfileEmailAddress';
```
Comment (PierrickVoulet) [inline examples/Utils/ArgumentNames.php:79]: This should be `BUSINESS_PROFILE_EMAIL` (same in the code example `AddBusinessProfileLocationExtensions`)

File: examples/AdvancedOperations/AddSmartCampaign.php [hunk 3]
```diff
@@ -88,11 +88,11 @@ class AddSmartCampaign
     // typically not recommended because they are less effective than suggested keyword themes,
     // however they are useful in situations where a very specific term needs to be targeted.
     private const FREEFORM_KEYWORD_TEXT = null;
-    // Optional: Specify the ID of a Google My Business (GMB) location. This is required if a
+    // Optional: Specify the ID of a Business Profile location. This is required if a
     // business name is not provided. It can be retrieved using the GMB API, see:
```
Comment (PierrickVoulet) [inline examples/AdvancedOperations/AddSmartCampaign.php:92]: Should change for `Business Profile API`

**Review by PierrickVoulet — CHANGES_REQUESTED** [review]

File: examples/AdvancedOperations/AddSmartCampaign.php (hunk 3 above)
Comment (wfansh) [inline examples/AdvancedOperations/AddSmartCampaign.php:92]: Business Profile API

File: examples/Utils/ArgumentNames.php (hunk 2 above)
Comment (wfansh) [reply examples/Utils/ArgumentNames.php:79]: +1

File: examples/Extensions/AddBusinessProfileLocationExtensions.php [hunk 4]
```diff
@@ -67,8 +67,8 @@ public static function main()
         // into the constants above.
         $options = (new ArgumentParser())->parseCommandArguments([
             ArgumentNames::CUSTOMER_ID => GetOpt::REQUIRED_ARGUMENT,
-            ArgumentNames::GMB_EMAIL_ADDRESS => GetOpt::REQUIRED_ARGUMENT,
-            ArgumentNames::GMB_ACCESS_TOKEN => GetOpt::REQUIRED_ARGUMENT,
+            ArgumentNames::BUSINESS_PROFILE_EMAIL_ADDRESS => GetOpt::REQUIRED_ARGUMENT,
```
Comment (wfansh) [inline examples/Extensions/AddBusinessProfileLocationExtensions.php:70]: Should be "BUSINESS_PROFILE_EMAIL"

**Review by wfansh — CHANGES_REQUESTED** [review]

**Review by wfansh — APPROVED** [review]

**Review by PierrickVoulet — APPROVED** [review]

**Pull request merged**